jre_emul/jre_sources.mk

```diff
@@ -1680,6 +1680,7 @@ JAVA_PRIVATE_SOURCES_ZIP = \
   java/util/zip/ZStreamRef.java \
   java/util/zip/ZipCoder.java \
   java/util/zip/ZipConstants64.java \
+  java/util/zip/ZipUtils.java \
```

Comment:
Add com/google/j2objc/nio/file/attribute/FileTime.java to JAVA_PRIVATE_SOURCES_ZIP. We'll move it as needed if other classes reference it.

Reply:
I've added this file in JAVA_PRIVATE_SOURCES_ZIP in my commit ccb13028f3ea1a603ba60a7cab486e33fddfdc0f on line 1679.